test/amazonica/test/ecs.clj

```diff
@@ -0,0 +1,7 @@
+(ns amazonica.test.ecs
+  (:require [amazonica.aws.ecs :refer :all]
+            [clojure.set :as set]
+            [clojure.test :refer :all]))
+
+
+
```

Comment:
Imy use case was to perform a rolling restart which I intended to use as the example, but it snowballed into more code then I expected. I can add some basic examples of listing services or something though.